main_test.go

```diff
@@ -1,15 +1,13 @@
 package main
 
 import (
-	"bytes"
 	"testing"
 
 	"foxygo.at/evy/assert"
 )
 
-func TestMain(t *testing.T) {
-	got := &bytes.Buffer{}
-	err := compile(got)
-	assert.NoError(t, err)
-	assert.Equal(t, "🌱\n", got.String())
+func TestTruncate(t *testing.T) {
+	assert.Equal(t, "123", truncate("12345", 3))
+	assert.Equal(t, "12345", truncate("12345", 5))
+	assert.Equal(t, "12345", truncate("12345", 6))
```

Comment:
ideally they'd be a test with a multi-byte character (`🌱🌱🌱🌱🌱` perhaps) to make sure it does that properly since the point of the function is to handle that properly.

Reply:
done.

Reply:
Not sure how I feel about the truncate failing now. What if it cuts a fox in half?